openmdao/recorders/sqlite_recorder.py

```diff
@@ -198,6 +198,10 @@ def _initialize_database(self):
                 if rank == 0:
                     metadata_filepath = f'{self._filepath}_meta'
                     print(f"Note: Metadata is being recorded separately as {metadata_filepath}.")
+                    try:
+                        os.remove(metadata_filepath)
```

Comment:
I wonder if we should at least throw a warning that there is an existing file being overwritten...

Reply:
I was wondering that too. Probably would be nice. We should do the same for the case files also, correct? We don't do that now.

Reply:
ya, good point...   maybe we should have a warning both here and in the similar code block in SqliteRecorder:
https://github.com/OpenMDAO/OpenMDAO/blob/master/openmdao/recorders/sqlite_recorder.py#L211

Reply:
Good idea! Will do